astroquery/heasarc/tests/test_heasarc_remote_isdc.py

```diff
@@ -162,9 +162,10 @@ def test_mission_list(self):
         with self.isdc_context:
             missions = heasarc.query_mission_list()
 
-        # Assert that there are indeed a large number of tables
-        # Number of tables could change, but should be > 900 (currently 956)
-        assert len(missions) == 5
+        # Assert that there are indeed numerous tables from ISDC
+        # Qunatity of tables could change due to breakdown of the descriptions
+        # Accessing via isdc returns only Integral related tables from HEASARC
+        assert len(missions) >= 5
```

Comment:
I made this change to `test_mission_list` since it wasn't passing and the previous comment was inaccurate. In short, accessing heasarc from isdc will just return stuff related to the integral mission. The 5 tables that were expected before have now grown to 8 since some have been added with proposal and description info. 

I just followed the lead of the heasarc remote tests and just asserted that as long as we are getting multiple tables (from what was previously seen), that should be ok.